src/library/Uno.Themes.WinUI.Markup/Theme/Typography.cs

```diff
@@ -0,0 +1,221 @@
+﻿using System;
+using Microsoft.UI.Xaml.Media;
+using Uno.Extensions.Markup.Internals;
+using Windows.UI.Text;
+
+namespace Uno.Themes.Markup
+{
+	public static partial class Theme
+	{
+		public static class Typography
+		{
+			public static class Resources
+			{
+				public static class DisplayLarge
+				{
+					[ResourceKeyDefinition(typeof(FontWeight), "DisplayLargeFontWeight")]
```

Comment:
Font-related resources should go into a `Typography`

Reply:
These are special cases for Typography. So it makes sense to bundle the font resources together by font style.